pymc3/distributions/distribution.py

```diff
@@ -176,7 +176,23 @@ def __init__(self, shape=(), dtype=None, defaults=('median', 'mean', 'mode'),
 
 
 class DensityDist(Distribution):
-    """Distribution based on a given log density function."""
+    """Distribution based on a given log density function.
+        
+        A distribution with the passed log density function is created, 
```

Comment:
That comma should be a period, and "requires" on the next line should then get capitalized ("Requires")

Reply:
That was intentional, but I see your point. It makes more sense this way!